assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -239,6 +239,9 @@ module.exports = elementorModules.ViewModule.extend( {
 			elementorCommon.helpers.hardDeprecated( 'elementorFrontend.utils.lightbox.setHTMLContent', '3.1.4' );
 		}
 
+		// Sanitize the HTML.
+		html = html.replace( /<script>|<\/script>/gmi, '' );
```

Comment:
It will not work for `<scr<script>ipt>`, after run `html.replace( /<script>|<\/script>/gmi, '' )` it will be `<script>`

Reply:
Done, @matipojo delete comment